Arm/rover_arm_system.hpp

```diff
@@ -34,18 +34,26 @@ class RoverArmSystem : public sjsu::common::RoverSystem
   };
   struct MissionControlData : public RoverMissionControlData
   {
-    enum class Modes : char
+    enum class ArmModes : char
     {
       kHomeArm    = 'A',
       kHomeHand   = 'H',
       kConcurrent = 'C',
       kRotunda    = 'R',
       kShoulder   = 'S',
       kElbow      = 'E',
-      kWrist      = 'W'
-
+      kHand       = 'D'
+    };
+    ArmModes ArmMode = ArmModes::kConcurrent;
```

Comment:
```suggestion
    ArmModes ArmMode = ArmModes::kConcurrent;
```
move this line down below with the others, almost didn't realize it was still here.

Reply:
got it